llvm/lib/Target/AIE/AIE2Schedule.td

```diff
@@ -462,11 +468,35 @@ MemInstrItinData<II_LDA_R,
 	EmptyCycles<4>, SimpleCycle<R_WA_PORT>], [7,1,1],
     MemoryCycles<[5]>>,
 MemInstrItinData<II_LDA_POST_1D,
-    [AvoidPartWordStore, EmptyCycles<1>, SimpleCycle<LOAD_UNIT_A>, EmptyCycles<4>,
+    [AvoidPartWordStore, SimpleCycle<P_WM_PORT>, SimpleCycle<LOAD_UNIT_A>, EmptyCycles<4>,
```

Comment:
Hi @gbossu, @andcarminati , please review the modified schedule for II_LDA_POST_1D,
based on my understanding of ISA we will update the pointer reg in E1 stage and thus will use the write port in E1 also.